### PR TITLE
fix Bug and remove some warnings

### PR DESCRIPTION
**Pull request by marc0090**

### CODE REVIEW

File: src/VisualStudioUI.VSMac/Options/TextOptionVSMac.cs [hunk 1]
```diff
@@ -102,7 +102,10 @@ private NSMenu CreateMenu()
                 else
                 {
                     NSMenuItem menuItem = new NSMenuItem();
-                    menuItem.Title = item.Label;
+                    string title = item.Label;
+                    //for some unknow reason to remove all "_" in titles,to be consistent with old code
```
Comment (BretJohnson) [inline src/VisualStudioUI.VSMac/Options/TextOptionVSMac.cs:106]: Do you know where this happened in the old code? I'd prefer if the shared visual studio ui code didn't have this logic. Instead, can we just remove the "_" in the vsmac-xamarin-extensions code somewhere, maybe just updating the title strings?

Comment (marc0090) [reply src/VisualStudioUI.VSMac/Options/TextOptionVSMac.cs:106]: yes, it also can be removed from vsmac-xamarin-extensions code

**Review by MMooreGitHub — APPROVED** [review]
Drive or provide assistance

**Review by MMooreGitHub — APPROVED** [review]
Looks good, overall.

**Review by TL-2015 — APPROVED** [review]

**Pull request merged**